File: In_C_Programming/q14.c
#include <stdio.h>

void count_Values(int arr[], int ln)
{

    int zero = 0, pos = 0, neg = 0;

    for (int i = 0; i < ln; i++)
    {
        if (arr[i] == 0)
        {
            zero++;
        }
        else if (arr[i] > 0)
        {
            pos++;
        }
        else
        {
            neg++;
        }
    }

    printf("The Count of Positive Values is: %d", pos);
    printf("The Count of Negetive Values is: %d", neg);
    printf("The Count of Zero Values is: %d", zero);
}

void get_Element(int arr[], int ln)
{

    printf("Enter The Elements: \n");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", &arr[i]);
    }
}

int main()
{

    int size;

    printf("Enter The Aray Length: \n");
    scanf("%d", &size);

    int arr[size];

    get_Element(arr, size);
    count_Values(arr, size);

    return 0;
}

File: In_Java_Programming/q14.java
import java.util.Scanner;
import java.util.Vector;

public class q14 {

    public static void get_Count(Vector<Integer> vec, int ln, int zero, int pos, int neg) {

        for (Integer i : vec) {

            if (i < 0) {
                neg++;
            } else if (i > 0) {
                pos++;
            } else {
                zero++;
            }
        }

        System.out.println("The Count of Positive Integers are: " + pos);
        System.out.println("The Count of Negetive Integers are: " + neg);
        System.out.println("The Count of Zero Integers are: " + zero);
    }

    public static void get_Element(Vector<Integer> vec, int ln, Scanner sc) {
        System.out.println("Enter The array Elements: ");

        for (int i = 0; i < ln; i++) {
            int val = sc.nextInt();
            vec.addLast(val);
        }
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter Array Length: ");
        int ln = sc.nextInt();

        Vector<Integer> vec = new Vector<Integer>();

        get_Element(vec, ln, sc);

        int zero = 0, pos = 0, neg = 0;

        get_Count(vec, ln, zero, pos, neg);

        sc.close();
    }
}
